### ISSUE TITLE
Place Package.resolved into .gitignore file

### ISSUE THREAD

**Issue by eviltofu** [AlphaWallet/alpha-wallet-ios#4099]
Between Xcode 13.2.1 and 13.3, the format for Package.resolved changed from version 1 to version 2. This change is not backward compatible. By placing Package.resolved into the .gitignore file, we can workaround this issue.

**Comment by eviltofu** [comment]
@hboon I installed `Xcode-install` via ruby gem in my `rvm` environment. Then I installed Xcode 13.2.1 alongside my 13.3. I deleted the `Package.resolved` file from `AlphaWallet.xcworkspace/xcshareddata/swiftpm` and place `Package.resolved` into the `.gitignore` file. I ran the `make clean && make install_bundle && make bootstrap && make test` under Xcode 13.2.1 and 13.3. The newly created Package.resolved file was not checked in by git. 

Should I go ahead and do a pull request for this change?
